src/lib/services/transfer/__tests__/access.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { access } from '../index';
+import { createStorage } from '../../../core/storage';
+
+import { mirror } from '../../../../__mocks__/handlers';
+
+describe('task', () => {
+  beforeEach(() => {
+    createStorage('memory');
+  });
+
+  test('getAll', async () => {
+    const {
+      req: { url, method, headers },
+    } = await mirror(await access.getAll('example-endpoint-id'));
+    expect({
+      url,
+      method,
+      headers,
+    }).toMatchSnapshot();
+  });
+
+  test('get', async () => {
+    const {
+      req: { url, method, headers },
+    } = await mirror(
+      await access.get({ endpoint_xid: 'example-task-id', id: 'example-access-rule-id' }),
```

Comment:
```suggestion
      await access.get({ endpoint_xid: 'example-endpoint-id', id: 'example-access-rule-id' }),
```

^ The current implementation makes the snapshots look incorrect.